src/jbmc/jbmc_parse_options.cpp

```diff
@@ -865,6 +1003,8 @@ void jbmc_parse_optionst::help()
     // This one is handled by jbmc_parse_options not by the Java frontend,
     // hence its presence here:
     " --java-unwind-enum-static    try to unwind loops in static initialization of enums\n" // NOLINT(*)
+    // Currently only supported in the JBMC frontend:
+    " --symex-driven-lazy-loading  only load functions when first entered by symbolic execution\n" // NOLINT(*)
```

Comment:
This should be defined in a header file to allow re-use across different driver programs

Reply:
Hmmm, it is pretty jbmc-specific at the moment though. I can't see anything except the option name being shareable with test-gen at the moment -- and it surely doesn't belong to bmct or goto_symext, both of which don't care exactly how their get_goto_program function works.

In short I agree some sort of factoring is in order at some point, but I can't see how to do it just now.

Reply:
But maybe in future CBMC will care?

Reply:
This is possible. For now CBMC uses none of the lazy loading infrastructure at all, and I believe @kroening's plan is for it to stay that way for a while. If and when it does, some factoring will be in order.